src/scss/_mixins.scss

```diff
@@ -24,6 +19,14 @@
 		$weight: regular;
 	}
 
+	// Check the font is an Origami font, not a custom font.
+	@if map-has-key($o-fonts-families, $family) {
+		$font-family-config: map-get($o-fonts-families, $family);
+		@if map-get($font-family-config, 'custom') {
+			@error 'Can not include a custom font "#{$family}". Include your custom font manually.';
```

Comment:
Does this error condition make this a breaking change?

Reply:
No as the 'custom' property in `$font-family-config` is new, so it should not ever error for existing cases.